lib/compiler/src/compile.erl

```diff
@@ -795,15 +800,19 @@ standard_passes() ->
 
 abstr_passes(AbstrStatus) ->
     case AbstrStatus of
-        non_verified_abstr -> [{unless, no_lint, ?pass(lint_module)}];
+        non_verified_abstr -> [{unless, no_lint, ?pass(lint_module)},
+                               {iff,beam_docs,?pass(beam_docs)}];
         verified_abstr -> []
     end ++
         [
+         ?pass(remove_doc_attributes),
```

Comment:
We should only run the remove_doc_attr pass in the `non_verified_abstr` case as otherwise we will run it twice.